buildSrc/src/main/groovy/io.micronaut.build.internal.convention-quality.gradle

```
@@ -5,7 +5,7 @@ plugins {
 }
 
 tasks.named("htmlSanityCheck") {
-    sourceDir = new File("${rootProject.buildDir}/docs/guide/")
+    sourceDir = file("${rootProject.layout.buildDirectory.asFile.get()}/docs/guide/")
```

:x: This is a mix of `file` and `layout`.
```suggestion
        sourceDir = rootProject.layout.buildDirectory.dir("docs/guide").get().asFile
```

```

---

fixed